wolfcrypt/src/port/iotsafe/iotsafe.c

```diff
@@ -719,6 +719,79 @@ static int iotsafe_put_public_key(byte *pubkey_id, unsigned long id_size,
     }
     return ret;
 }
+#ifdef HAVE_HKDF
+static int iotsafe_hkdf_extract(byte* prk, const byte* salt, word32 saltLen,
+       byte* ikm, word32 ikmLen, int digest)
+{
+    int ret;
+    char *resp;
+	uint16_t hash_algo  = 0;
+    int len;
+
+    WOLFSSL_MSG("Enter iotsafe_hkdf_extract");
+     switch (digest) {
+        case WC_SHA256:
```

Comment:
wrap the case..break within `#ifndef NO_SHA256`

Reply:
done